src/components/ddo-searchbar.tsx

```diff
@@ -9,7 +9,7 @@ import classnames from "classnames";
 /** The URL for Data-Driven Onboarding (DDO) on the JustFix Tenant Platform. */
 const DDO_URL =
   (process.env.GATSBY_TENANT_PLATFORM_SITE_ORIGIN ||
-    "https://demo.justfix.nyc") + "/ddo";
```

Comment:
@toolness this ddo pathname addition seems to be a relic from olden days, right? Good to remove?

Reply:
Yeah, looks like it's from our ancient days.  Might be useful to still provide a fallback, but a newer one?  Otherwise I'm worried that new devs who get the site running might not be able to use DDO w/o explicitly setting `GATSBY_TENANT_PLATFORM_SITE_ORIGIN `, which would be a bummer.  But if that env var already has a default value in our config files then should be fine to remove this entirely.

Reply:
oops sorry I sent that message before seeing what you had replaced it with!  Yeah this seems fine, ignore my previous comment!